src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -912,6 +922,75 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         // not working properly.
         manager.register_pass<ov::pass::Validate>();
 
+        manager.register_pass<ov::pass::RoPEFusion>(true);
+        pass_config->disable<ov::pass::RoPEFusionGPTJ>();
+        pass_config->disable<ov::pass::RoPEFusionIOSlicing>();
+        pass_config->disable<ov::pass::RoPEShareCosSin>();
+
+        float activations_scale_factor = config.get_property(ov::hint::activations_scale_factor);
+        ov::element::Type scaled_precision = element::f16;
+
+        if (activations_scale_factor > 0.f) {
+            using namespace ov::pass::low_precision;
+
+            auto supportedPrecisions = std::vector<PrecisionsRestriction>({});
+            auto perTensorQuantization = std::vector<QuantizationGranularityRestriction>({});
+
+            pass_config->disable<ov::pass::AddMultiplyFusion>();
+            pass_config->disable<RecurrentCellTransformation>();
+            pass_config->disable<MultiplyToGroupConvolutionTransformation>();
+            pass_config->disable<ConvolutionTransformation>();
+            pass_config->disable<ConvolutionBackpropDataTransformation>();
+            pass_config->disable<GroupConvolutionTransformation>();
+            pass_config->disable<MatMulTransformation>();
+            pass_config->disable<MVNTransformation>();
+            pass_config->disable<ConcatTransformation>();
+            pass_config->disable<ClampTransformation>();
+            pass_config->disable<UnsqueezeTransformation>();
+            pass_config->disable<VariadicSplitTransformation>();
+            pass_config->disable<ReshapeTransformation>();
+            pass_config->disable<TransposeTransformation>();
```

Comment:
Could you please elaborate why do we disable all these LPT passes?

Some of them are duplicated by new activations scaling transformations (e.g. `MulDownTransformation` seems like `ReshapeTransformation` and `TransposeTransformation` duplication). At first glance, we can avoid the new code here

Reply:
`AddMultiplyFusion` was disabled because it converts fc --> add --> mul to fc --> mul --> add.

`ConvolutionTransformation`, `ConvolutionBackpropDataTransformation`, `GroupConvolutionTransformation` and `MatMulTransformation` were disabled because `ScaleDownSingleLayer` adds multiply layers around Conv and MatMul.

`UnsqueezeTransformation` and `VariadicSplitTransformation` make some eltwise layers to run in fp32 precision, even if infer precision if fp16. This situation can be checked with the newly added functional test.

`ReshapeTransformation` and `TransposeTransformation` were re-enabled.